<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="produto">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>1120</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 italic 25pt &quot;Courier&quot;;
background-color: rgb(253, 255, 216);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="empresa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 81 10pt &quot;Rockwell Extra Bold&quot;;

background-color: rgb(223, 213, 255);

border-color: rgb(162, 162, 162);

border-style: Outset;
border-width: 2px;
border-radius: 12px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../Users/Helio/.designer/backup/CADASTRO.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>420</width>
      <height>460</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 255, 216);
color: rgb(0, 0, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_fecha">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>600</y>
      <width>220</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;F10&gt;FECHAMENTO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>100</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 18pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Cliente:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sub_total">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>420</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 255, 216);
font: 700 19pt &quot;Courier&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>SUB-TOTAL:</string>
    </property>
   </widget>
   <widget class="QLabel" name="numero_pedido">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>420</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 255, 216);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cabecalho">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>420</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(253, 255, 216);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_cliente">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>100</y>
      <width>510</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Courier&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 255);
font: 700 20pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>DINHEIRO:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_dinheiro">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>180</y>
      <width>290</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>26</pointsize>
      <italic>false</italic>
      <bold>true</bold>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 26pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="wrapping">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="maximum">
     <double>999999.989999999990687</double>
    </property>
    <property name="stepType">
     <enum>QAbstractSpinBox::DefaultStepType</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>250</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 255);
font: 700 20pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>PIX:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_pix">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>250</y>
      <width>290</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 26pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <double>999999.989999999990687</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>320</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 255);
font: 700 20pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>CARTAO:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_cartao">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>320</y>
      <width>290</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 26pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <double>999999.989999999990687</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>390</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 255);
font: 700 20pt &quot;Segoe UI&quot;;
</string>
    </property>
    <property name="text">
     <string>DUPLICATA:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_duplicata">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>390</y>
      <width>290</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 26pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <double>999999.989999999990687</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>460</y>
      <width>180</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 0, 255);
font: 700 20pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>CHEQUE:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ds_cheque">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>460</y>
      <width>290</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 26pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="specialValueText">
     <string/>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="maximum">
     <double>999999.989999999990687</double>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_sair">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>600</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_dinheiro">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>190</y>
      <width>210</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 italic 25pt &quot;Courier&quot;;
color: rgb(0, 170, 0);
background-color: rgb(253, 255, 216);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_pix">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>260</y>
      <width>210</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 italic 25pt &quot;Courier&quot;;
color: rgb(0, 170, 0);
background-color: rgb(253, 255, 216);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_cartao">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>330</y>
      <width>210</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">ffont: 700 italic 25pt &quot;Courier&quot;;
color: rgb(0, 170, 0);
background-color: rgb(253, 255, 216);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_duplicata">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>400</y>
      <width>210</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 italic 25pt &quot;Courier&quot;;
color: rgb(0, 170, 0);
background-color: rgb(253, 255, 216);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lb_cheque">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>470</y>
      <width>210</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 italic 25pt &quot;Courier&quot;;
color: rgb(0, 170, 0);
background-color: rgb(253, 255, 216);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>cb_cliente</tabstop>
  <tabstop>ds_dinheiro</tabstop>
  <tabstop>ds_pix</tabstop>
  <tabstop>ds_cartao</tabstop>
  <tabstop>ds_duplicata</tabstop>
  <tabstop>ds_cheque</tabstop>
  <tabstop>bt_fecha</tabstop>
  <tabstop>bt_sair</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
